Tests/ViewInspectorTests/SwiftUI/TreeViewTests.swift

```diff
@@ -28,6 +28,16 @@ final class TreeViewTests: XCTestCase {
         }
         XCTAssertEqual(sut.content.medium.viewModifiers.count, count)
     }
+
```

Comment:
I am having trouble understanding how TreeView.swift had 0% code coverage given that the above tests exist.

Reply:
Overall I'm wondering if this TreeView.swift file is related to a different type of "tree"? I'm trying to assess if we should be adding these new tests to this or a new file.

Reply:
That's a great question! I debugged the existing tests and none of them hit the code in the TreeView.swift file.
I enabled coverage to double check that none of the other tests were exercising this code path.

@nalexn any thoughts this?

Reply:
If you run the tests on earlier os versions (I have 14.4 simulator image installed), that code would get executed. SwiftUI internals get a refactor with every major os version, so we have to keep the old parsers in place to support previous versions.